megamek/src/megamek/client/bot/BotClient.java

```diff
@@ -113,7 +113,7 @@ public void run() {
     }
 
     public BotClient(String playerName, String host, int port) {
-        super(playerName, host, port);
+        super(playerName, true, host, port);
```

Comment:
The other way you could implement the isBot() property is just having the BotClient class override it and return a fixed true, while the regular Client returns a fixed false. That way you don't have to modify the constructor and store the value.

Reply:
Good point, ta!

Fixed.